app/controllers/forms_controller.rb

```diff
@@ -43,6 +43,18 @@ def update
     end
   end
 
+  def new
+    form = Form.create! name: 'new-form'
```

Comment:
Currently working on getting around this.

So the correct "Rails" way to do this is to take the edit page for the draft that we have and just display it in both `form_drafts/edit` and `forms/new`. It's a partial in `forms/_form`. Cool. We made a `new` controller action that just does this now:
``` ruby
  form = form.new
  @draft = form.create_draft
```
Problems have arisen.

The form on the `forms/new` page is trying to post to a create action in the form_drafts controller. There is none. In `form_drafts/edit`, it posts to the `update` action. Cool, but we can't have it do that now, because it's not editing an existing `form_draft`. It's actually (it being `form_for`) trying to create a path_helper for the create action in form_drafts, and there is no create action in form_drafts.

Could make a create action in form_drafts, but would have to create a form in the same action. And then in that action, also delete the form_drafts. This feels even more wrong than what's going on now, which is just that we're trying to get around name uniqueness validation.

I do not see a way around this except to rewrite the way forms and their drafts are handled, OR to just create a unique placeholder name for a new form when it's created. Currently, the issue is that 'new-form' will be the new name of any subsequent forms created, so someone could in theory break the application. We can change this fact in one line.

Either way is fine, but then that quickly becomes something I would not have assigned a new programmer.